lib/matplotlib/dates.py

```diff
@@ -1175,6 +1175,15 @@ def __call__(self):
         return self.tick_values(dmin, dmax)
 
     def tick_values(self, vmin, vmax):
+        start, stop = self._create_rrule(vmin, vmax)
+        dates = self.rule.between(start, stop, True)
+        if len(dates) == 0:
+            return date2num([vmin, vmax])
```

Comment:
Yes can you add a test for this.  Otherwise I think this is good?

Reply:
@jklymak done